examples/lock-app/nrf5/main/AppTask.cpp

```diff
@@ -222,10 +231,62 @@ void AppTask::HandleBLEMessageReceived(chip::Ble::BLEEndPoint * endPoint, chip::
     chip::System::PacketBuffer::Free(buffer);
 }
 
+void SendUDPBroadCast()
```

Comment:
nit: How about we make a common function `void SendUDPBroadcast(const char *aName)` for both lock and light App?

Reply:
Done by move this to Server.cpp